.github/workflows/auto-assign-reviewer.yaml

```diff
@@ -9,8 +9,10 @@ permissions:
 
 jobs:
   auto-assign:
+    if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     steps:
-    - uses: rancher/gh-issue-mgr/auto-assign-action@main
-      with:
-        configuration-path: .github/auto-assign-config.yaml
\ No newline at end of file
+      - uses: rancher/gh-issue-mgr/auto-assign-action@main
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The action supports auto assign the author as an assignee, but it doesn't seem to be working at the moment, not sure if this fix will resolve the issue.

Reply:
Looks like it's working.
https://github.com/harvester/harvester-ui-extension/actions/runs/16158741422/job/45606197638?pr=382#step:2:7